test/smoke/check_smoke.sh

```diff
@@ -242,12 +242,17 @@ if [ "$AOMP_PARALLEL_SMOKE" == 1 ]; then
         sem --jobs 4 --id def_sem -u 'base=$(basename $(pwd)); make check > /dev/null; if [ $? -ne 0 ]; then flock -e lockfile -c "echo $base: Make Failed >> ../make-fail.txt"; fi;'
       elif [ $base == "gpus" ]; then # Compile and link only test
         echo gpus is compile only!
+      elif [ "$AOMP_SANITIZER" == 1 ]; then
+        sem --jobs 4 --id def_sem -u 'make check-asan > /dev/null 2>&1'
```

Comment:
I don't think we need to call the check-asan target for tests outside of smoke-asan. Normal smoke tests can still use make-check.